2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -17,69 +17,54 @@ import sbt._
 object Dependencies {
 
   val resolutionRepos = Seq(
-    "Snowplow Analytics Maven repo"          at "http://maven.snplow.com/releases/",
-    "Snowplow Analytics Maven snapshot repo" at "http://maven.snplow.com/snapshots/",
-    "Spray repo"                             at "http://repo.spray.io",
-    "Typesafe repository"                    at "http://repo.typesafe.com/typesafe/releases/",
-    // For Scalazon
-    "BintrayJCenter"                         at "http://jcenter.bintray.com",
-    // For sbt-thrift
-    "bigtoast-github"                        at "http://bigtoast.github.com/repo/"
+    "Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/",
+    // For uaParser utils
+    "user-agent-parser repo" at "https://clojars.org/repo/"
```

Comment:
I think the first one can be removed, however I think the ua parser is needed if you haven't compiled sce before? I don't think resolvers are transitives.

Reply:
Apparently I had check that since both are needed:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.clojars.timewarrior#ua-parser;1.3.0: not found
[warn] 	:: com.snowplowanalytics#snowplow-thrift-raw-event;0.1.0: not found
[warn] 	:: com.snowplowanalytics#collector-payload-1;0.0.0: not found
[warn] 	:: com.snowplowanalytics#schema-sniffer-1;0.0.0: not found
[warn] 	:: com.snowplowanalytics#referer-parser_2.11;0.3.0: not found
[warn] 	:: com.snowplowanalytics#scala-maxmind-iplookups_2.11;0.3.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

Reply:
Ok. Seems these were cached for me (though quite unexpected since I've never compiled SCE on this laptop)